lib/extend/tag.js

```diff
@@ -158,14 +158,17 @@ const getContext = (lines, errLine, location, type) => {
  * @param  {string}   str string input for Nunjucks
  * @return {Error}    New error object with embedded context
  */
-const formatNunjucksError = (err, input) => {
+const formatNunjucksError = (err, input, source = '') => {
   const match = err.message.match(/Line (\d+), Column \d+/);
   if (!match) return err;
   const errLine = parseInt(match[1], 10);
   if (isNaN(errLine)) return err;
 
   // trim useless info from Nunjucks Error
-  const splited = err.message.replace('(unknown path)', '').split('\n');
+  const splited = err.message.replace('(unknown path)', () => {
+    if (source) return magenta(source);
+    return '';
+  }).split('\n');
```

Comment:
Instead of a function, static string will be better.
```js
const formatedSource = source ? magenta(source) : '';
const splited = err.message.replace('(unknown path)', formatedSource).split('\n');
```